tools/benchmarksAnalyzer/src/main/kotlin/org/jetbrains/kliopt/KliOpt.kt

```diff
@@ -0,0 +1,225 @@
+/*
+ * Copyright 2010-2018 JetBrains s.r.o.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+
+package org.jetbrains.kliopt
+
+import org.jetbrains.benchmarksAnalyzer.exitProcess
```

Comment:
Nope, better not use exitProcess in relatively general purpose library code.

Reply:
Sorry, I a bit don't understand. What then is better way to exit program after using help flag? All known by me arg parsers added help flag and behaviour for it by themselves.

Reply:
Argument parser shall not exit an application on its own, API consumer shall decide what to do depending on available args.